amundsen_application/static/js/ducks/tableMetadata/api/v0.ts

```diff
@@ -144,8 +144,12 @@ export function getPreviewData(queryParams: PreviewQueryParams) {
     return { data: response.data.previewData, status: response.status };
   })
   .catch((e: AxiosError<PreviewDataAPI>) => {
-    const data = e.response ? e.response.data.previewData : {};
-    const status = e.response ? e.response.status : null;
+    const response = e.response;
+    let data = {};
+    if (response && response.data && response.data.previewData) {
```

Comment:
Why we need optional chaining https://github.com/tc39/proposal-optional-chaining